docs/runtime_options.md

```diff
@@ -134,7 +134,85 @@ cAdvisor stores the latest historical data in memory. How long of a history it s
 --storage_driver_user="root": database username (default "root")
 ```
 
-For storage driver specific instructions:
+## Perf Events
+
+```
+--perf_events_config="" Path to a JSON file containing configuration of perf events to measure. Empty value disables perf events measuring.
+```
+
+See example configuration below:
+```json
+{
+  "non_grouped": [
+    {
+      "type": 4,
+      "config": [
+        "0xc0"
+      ]
+    } 
+  ],
+  "grouped": [
+    [
+      {
+        "type": 13,
+        "config": [
+          "0xc04"
+        ]
+      },
+      {
+        "type": 13,
+        "config": [
+          "0xc304"
+        ]
+      }
+    ]
+  ],
+  "interval": "5s"
+}
+```
+
+In the example above number:
+* `INST_RETIRED.ANY_P` (number of instructions retired on 2nd Generation Intel® Xeon® Scalable Processor) identified by
+`config[0xc0]` will be measured as non-grouped event
+* `UNC_M_CAS_COUNT.WR` (all DRAM Write CAS commands issued or `cas_count_write`) identified by `config[0xC04]` and 
+`UNC_M_CAS_COUNT.RD` (all DRAM Read CAS commands issued or `cas_count_read`) identified by `config[0xc304]` will be 
+measured as grouped events. In their case `type` attribute indicates memory controller.
+
+##### Translating event name to config values
+
+In order to determine what values must be used to configure particular event you should follow a procedure:
+1. Identify the event in `perf list` output. 
```

Comment:
Is this something cAdvisor could do for users?   Should our API be higher-level, or is that not possible?

Reply:
I will look into it. There are some libraries (in and out of kernel) that can help.